### PR TITLE
Fix IDE0044: MakeFieldReadonly part 6

### PR DESCRIPTION
**Pull request by xtqqczze**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0044

_Split from #13880 to ease review._

`src\Microsoft.PowerShell.CoreCLR.Eventing\`

### CODE REVIEW

File: src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs [hunk 1]
```diff
@@ -11,25 +11,25 @@ namespace System.Diagnostics.Eventing
     public struct EventDescriptor
     {
         [FieldOffset(0)]
-        private ushort _id;
+        private readonly ushort _id;
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs:14]: The struct could be readonly too.

Comment (xtqqczze) [reply src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs:14]: `readonly` structs were a new feature in C# 7.2 , so there are likely many structs we can mark `readonly`.

See also: https://devblogs.microsoft.com/premier-developer/the-in-modifier-and-the-readonly-structs-in-c/

Comment (iSazonov) [reply src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs:14]: I don't think there are many, but you can open a new tracking issue.

Comment (xtqqczze) [reply src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs:14]: Issue opened: #13915

**Review by iSazonov — APPROVED** [review]

File: src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs (hunk 1 above)
Comment (xtqqczze) [reply src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs:14]: @iSazonov I have PR for readonly structs: #13919

**Pull request merged**